templates/memoryusagegraph.mustache

```diff
@@ -27,33 +27,10 @@
 
     Context variables required for this template:
     * id - The id of the profile
-    * request - The request url (with pathinfo and parameters)
-    * sessionid - The session ID
-    * userid - ID of the user who loaded the page
-    * scripttype - The script type
-    * created - Unix timestamp of profile creation
-    * duration - Profile duration in seconds.
-    * resposnecode - HTTP response code
-    * cookies - Is cookies enabled?
-    * buffering - Is buffering enabled?
-    * fullname - Full nane of user represented by userid.
-    * userlink - Link to user's profile page, or null if no user.
-    * tabs - Tabs markup.
 
     Example context (json):
     {
         "id" : 3,
-        "request" : "/my/index.php/another/deeper/path?time=day&duration=week",
```

Comment:
why are these chunks deleted?

Reply:
Because the template in question does not use the context object described. For instance, this is the mem usage graph, but the request field is only used on the profile template.